docs/source/dashboard.rst

```diff
@@ -0,0 +1,130 @@
+Dashboard Diagnostics
+=====================
+
+Profiling parallel code can be challenging, but the :doc:`Dask distributed scheduler <scheduling>` 
+provides live feedback via its interactive dashboard. 
+
+## Include dashboard picture status page
+
+To be able to to use the dashboard diagnostics `Bokeh <https://docs.bokeh.org>`_ needs to 
+be installed. A link that redirects to the dashboard will prompt whenever the scheduler is 
+created, and you can access the link when you create a client.
```

Comment:
If you `conda install dask distributed` you get Bokeh out of the box, and if you `pip install dask distributed` the Dashboard page shows a helpful message.

<img width="402" alt="image" src="https://user-images.githubusercontent.com/1610850/152147456-1a0b2be1-fb32-49ee-8e16-889d43a64519.png">

So perhaps telling folks that they need Bokeh here is unnecessary complexity?